introduction.md

```diff
@@ -0,0 +1 @@
+Kon'nichiwa! My name is Alberto, I'm a Web Dev and my favourite codeschool path is Ruby
```

Comment:
This bot should be updated to see if the suggestion was already cumplimented... Maybe its just a matter of 'Favourite' vs 'favorite' ? Since it suggests with the american version and in Europe most of us had to learn english with british books and teachers it will be a common diff. Thanks